internal/operators/ocs/consts.go

```diff
@@ -3,9 +3,25 @@ package ocs
 type ocsDeploymentMode string
 
 const (
+	// Aggregate CPU for compact mode is 36(including OCP), OCP requires 4 CPU per host on master, and 2 CPU for each disk
+	// so per host requires (36-(4*3)-(2*3))/3 = 6 CPU per host
+	CPUCompactMode int64 = 6
+
+	// Aggregate CPU for minimal mode is 24(including OCP), OCP requires 2 CPU per host on worker, and 2 CPU for each disk
+	// so per host requires (24-(2*3)-(2*3))/3 = 4 CPU per host
+	CPUMinimalMode int64 = 4
```

Comment:
The minimal deployment uses 1 cpu per disk.

Reply:
@nbalacha , what should we modify here? Should we do (24-(2 * 3)-(1 * 3))/3 = 5 cpus, and (disks) * 1 cpu ?

Reply:
Based on discussions we are leaving this as it is for now.